arches/app/models/migrations/11869_rename_resxres_fields.py

```diff
@@ -0,0 +1,128 @@
+        ),
+        migrations.AlterField(
+            model_name="resourcexresource",
+            name="node",
+            field=models.ForeignKey(
+                blank=True,
+                db_column="nodeid",
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="resxres",
+                to="models.node",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="resourcexresource",
+            name="from_resource",
+            field=models.ForeignKey(
+                blank=True,
+                db_column="resourceinstanceidfrom",
+                db_constraint=False,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="from_resxres",
+                to="models.resourceinstance",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="resourcexresource",
+            name="from_resource_graph",
+            field=models.ForeignKey(
+                blank=True,
+                db_column="resourceinstancefrom_graphid",
+                db_constraint=False,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="from_resxres",
+                to="models.graphmodel",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="resourcexresource",
+            name="to_resource",
+            field=models.ForeignKey(
+                blank=True,
+                db_column="resourceinstanceidto",
+                db_constraint=False,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="to_resxres",
+                to="models.resourceinstance",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="resourcexresource",
+            name="to_resource_graph",
+            field=models.ForeignKey(
+                blank=True,
+                db_column="resourceinstanceto_graphid",
+                db_constraint=False,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="to_resxres",
+                to="models.graphmodel",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="resourcexresource",
+            name="tile",
+            field=models.ForeignKey(
+                blank=True,
+                db_column="tileid",
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="resxres",
+                to="models.tilemodel",
+            ),
+        ),
+        migrations.RemoveField(
+            model_name="resourcexresource",
+            name="datestarted",
+        ),
+        migrations.RemoveField(
+            model_name="resourcexresource",
+            name="dateended",
```

Comment:
I think removing these is good. They're really holdovers from the older generic resource relationship approach. People should use nodes to manage that data.